jsk_2016_01_baxter_apc/firmware/gripper_front_manager_right.ino

```diff
@@ -50,7 +50,7 @@ inline float rad_to_deg(float rad)
 
 void set_angle(void)
 {
-    myservo.write((int)(-rad_to_deg(servo_angle_msg.data) + 90.0));
+    myservo.write((int)(rad_to_deg(servo_angle_msg.data) - 90.0));
```

Comment:
Is this for gripper-v3?
I think this shouldn't be fixed until right gripper becomes gripper-v3